scaleway/resource_lb_backend.go

```diff
@@ -237,20 +249,24 @@ func resourceScalewayLbBackend() *schema.Resource {
 }
 
 func resourceScalewayLbBackendCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
-	lbAPI := lbAPI(meta)
+	lbAPI, defaultZone, err := lbAPIWithZone(d, meta)
 
-	region, LbID, err := parseRegionalID(d.Get("lb_id").(string))
+	zone, LbID, err := parseZonedID(d.Get("lb_id").(string))
 	if err != nil {
 		return diag.FromErr(err)
 	}
+	// set default zone
```

Comment:
lb_id should always be a localised it. So we should never encounter this condition